backend/src/models/constants.js

```diff
@@ -11,3 +11,17 @@ exports.MAX_OPTION = 3
 
 // Paging Constants
 exports.PAGE_SIZE = 10
+
+// Table Names
+exports.TABLE_NAMES = {
+    USERS: 'user',
+    LIKES: 'likes',
+    COURSES: 'course',
+    SUBJECTS: 'subjects',
```

Comment:
Should we make pluralisation consistent? Not sure what the usual style is.

Reply:
I basically just made this so that we could change it later to reflect any future change to table names. The keys at least are all plurals now.